### PR TITLE
Fix CS.

### PR DESCRIPTION
**Pull request by dereuromark**
cleanup

### CODE REVIEW

**Comment by dereuromark** [comment]
With this we should make a new minor, raising the min level to 3.7+ and using less deprecations.

File: composer.json [hunk 1]
```diff
@@ -17,10 +16,11 @@
         }
     ],
     "require": {
-        "cakephp/cakephp": "^3.5"
+        "cakephp/cakephp": "^3.7"
```
Comment (ADmad) [inline composer.json:19]: Why is bumping CakePHP version needed?

Comment (ADmad) [reply composer.json]: I have reverted back the Cake version bump. The couple of method exists check in helpers aren't really a problem.

**Comment by ADmad** [comment]
Thanks.

**Pull request merged**